core/templates/services/math-interactions.service.ts

```diff
@@ -122,6 +162,17 @@ export class MathInteractionsService {
         'Your answer contains an invalid term: ' + invalidIntegers[0]);
       return false;
     }
+    let invalidMultiTerms = expressionString.match(
+      /([a-zA-Z]\d)/g);
```

Comment:
Should the letter part include greek letters, and should the digit part include multiple digits?

(Maybe add test cases for these.)

Reply:
Yes, it should. Added check and test cases.